src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +242,58 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
 key/value can be used to distinguish
 different reconstruction algorithms (for example ones using motion correction).
 
+Structural MR images whose intensity is represented in a non-arbitrary scale
+constitute parametric maps. Currently supported parametric maps include:
+
+| **Name**                                     | `suffix`  | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|----------------------------------------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Longitudinal relaxation time map             | T1map     | In seconds (s). T1 maps are REQUIRED to use this suffix irrespective of the method they are related to. See [this interactive book on T1 mapping](https://qmrlab.org/t1_book/intro) for further reading on T1-mapping.                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I think "irrespective of the method used to generate them." would be clearer. Possibly even "regardless" instead of "irrespective".

Reply:
https://github.com/bids-standard/bids-specification/pull/690/commits/8f97e119ed0d73e33ed38cfcb538bffa5721fd83